config/routes.rb

```diff
@@ -1,12 +1,10 @@
 Rails.application.routes.draw do
 
-  get 'static_pages/home'
-
-  get 'static_pages/help'
-
-  get 'static_pages/about'
-
-  root 'sessions#new'
+  root 'static_pages#home'
+  get 'home' => 'static_pages#home'
+  get 'help' =>  'static_pages#help'
+  get 'about' => 'static_pages#about'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.